src/yaml/ulissewolf/Freddy Fazbear's Pizza/ulissewolf-freddy-fazbears-pizza.yaml

```diff
@@ -0,0 +1,86 @@
+group: "ulissewolf"
+name: "ulissewolf-freddy-fazbears-pizza"
+version: "1.0.0"
+subfolder: "600-civics"
+dependencies:
+- "bsc:mega-props-jenx-vol03"
+- "bsc:vip-girafe-carpack-vol01-vol02"
+- "bsc:vip-girafe-carpack-vol01-vol02-remastered"
+- "bsc:vip-girafe-carpack-vol03"
+- "bsc:vip-girafe-carpack-vol04"
+- "bsc:mega-props-cp-vol01"
+- "bsc:mega-props-cp-vol02"
+- "bsc:mega-props-jes-vol01"
+- "supershk:mega-parking-textures"
+
+info:
+  summary: "Freddy Fazbear's Pizza"
+  description: >
+    This prop pack is the beginning of a long series of prop packs inspired by the Five Nights At Freddy’s
```

Comment:
It's not a prop pack if it comes with lots and buildings though, is it? So what is it? A landmark? A CS$?

Generally, my strong recommendation is to package only one lot type. If the building clearly looks like a landmark, only include the landmark. If it works as a specific civic building type, include only the civic lot. If it looks more like a growable, include only the growable lot.

In this case, it looks more like a CS$$ growable than anything else, which should be installed into the `300-commercial` subfolder. The ploppable should not be installed.